Node/calling/tests/TextBot.js

```diff
@@ -1,65 +0,0 @@
-var assert = require('assert');
```

Comment:
There's no 'TextBot' in this application. This test suite appears to be a duplicate of the dialog tests in the 'botbuilder' app, so removing from here.